controller/konnect/ops/ops_controlplane.go

```diff
@@ -128,6 +128,7 @@ func setGroupMembers(
 	id string,
 	sdkGroups sdkops.ControlPlaneGroupSDK,
 ) error {
+	// REVIEW: should we add MembersReferenceResolved condition for CP groups with 0 members?
```

Comment:
I believe we can set this one with a message stating that no members set. Just to be coherent on the API level. WDYT?

Reply:
Added a `NoMembers` reason for such case.